css/style.css

```diff
@@ -1,32 +1,173 @@
 @import url("https://fonts.googleapis.com/css2?family=Playfair+Display:ital,wght@0,400;0,500;0,600;0,700;0,800;0,900;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
 @import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
 
-* {
-  margin: 0;
-  padding: 0;
+html {
   box-sizing: border-box;
-  outline: none;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
 }
 
 body {
-  font-size: 100%;
-  box-sizing: border-box;
   margin: 0;
   padding: 0;
+  line-height: 1.6;
+  font-family: "Poppins", sans-serif;
+  outline: none;
+  overflow: scroll;
+  font-size: 62.8%;
+}
+
+h1,
+h2 {
+  font-family: "Playfair Display", serif;
+  font-weight: normal;
 }
 
 header {
   display: flex;
-  height: 12vh;
-  justify-content: end;
-  background-image: url("landing.png");
-  background-size: auto;
-  background-repeat: no-repeat;
-  background-position: left top;
+  height: 10vh;
+  width: 100%;
+
+  /* position: static; */
+}
+
+nav {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: space-between;
+  width: inherit;
+  height: 14vh;
+}
+
+.ham {
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  cursor: pointer;
+  right: 5%;
+  transition: 0.2s 0.1s;
+  border: none;
+  background: transparent;
+  padding: 0;
+  top: 5%;
+  box-sizing: inherit;
+}
+
+span {
+  background-color: black;
+  display: flex;
+  height: 5px;
+  width: 3em;
+  margin: 5px auto;
+  border-radius: 2px;
+}
+
+.menu {
+  background-color: white;
+  color: #1a2236;
+  font-size: 2.5em;
+  list-style: none;
+  flex-direction: column;
+  text-align: center;
+  padding: 1vh 2.7em;
+  justify-content: center;
+  align-items: end;
+  height: 88vh;
+  width: 100%;
+  display: none;
+  position: absolute;
+  opacity: 1;
+  top: 9vh;
+}
```

Comment:
- [ ] Kindly make sure your design matches your preferred template design😉
- Your Design
 ![screenshot (11)](https://user-images.githubusercontent.com/44624138/146018299-600e07c7-0871-4709-82f4-f6db06530f3e.png)
- Figma design
![screenshot (10)](https://user-images.githubusercontent.com/44624138/146018484-745626ee-ff93-474d-a77f-cfd3c8f2d222.png)

Reply:
- [ ] Kindly make the menu fixed so it won't scroll like the rest of the page and please make sure when user clicks any option of the mobile menu, the mobile menu disappear. In your case in scroll down to the page but when you scroll up the mobile menu is still visible✌

Reply:
Done